kommunicate/src/main/java/io/kommunicate/zendesk/KmZendeskClient.java

```diff
@@ -106,6 +111,12 @@ public void update(ConnectionStatus connectionStatus) {
                 }
                 zendeskConnected = true;
                 observeChatLogs();
+                if(!messagesInBuffer.isEmpty()) {
```

Comment:
is this if required? if it is already initialzed with some empty collection in order to avoid null pointer exception error then i guess this if check is not required. for loop should get care of it.

Reply:
For loop will take care of it, remove the if statement